src/css/styles.scss

```diff
@@ -80,54 +80,55 @@ header {
   margin: 1.2% 0.5% 1.2% 0%;
   width: 100%;
 
-  #search-input {
-    background-color: #b6dcca;
-    background: -webkit-gradient( linear,left top, left bottom, color-stop(0, rgb(255, 255, 255)), color-stop(1, rgb(225,225,225)));
-    border-radius: 10px 0px 0px 10px;
-    border: none;
-    box-shadow: inset -3px -1px 20px 1px #5c7894;
-    color: $blue-background-color;
-    font-family: $default-body-font-family, sans-serif;
-    font-size: 42px;
-    height: 100%;
-    outline: none;
-    padding: 0% 2% 0% 0%;
-    text-align: right;
-    transition: .8s ease;
-    width: 100%;
-  }
+}
+#search-input {
+  background-color: #b6dcca;
+  background: -webkit-gradient( linear,left top, left bottom, color-stop(0, rgb(255, 255, 255)), color-stop(1, rgb(225,225,225)));
+  border-radius: 10px 0px 0px 10px;
+  border: none;
+  box-shadow: inset -3px -1px 20px 1px #5c7894;
+  color: $blue-background-color;
+  font-family: $default-body-font-family, sans-serif;
```

Comment:
This change to the font family was a great call. Calibri is way more Dyslexia friendly